source/mongodump.txt

```diff
@@ -870,35 +842,36 @@ Options
    definitions when :binary:`~bin.mongodump` applies to an entire instance
    and not just a specific database.
    
-
-
 .. option:: --excludeCollection=<string>
 
    Excludes the specified collection from the :program:`mongodump` output.
    To exclude multiple collections, specify the :option:`--excludeCollection` multiple times.
 
-
 .. option:: --excludeCollectionsWithPrefix=<string>
 
    Excludes all collections with a specified prefix from the :program:`mongodump`
    outputs. To specify multiple prefixes, specify the :option:`--excludeCollectionsWithPrefix` multiple
    times.
 
-
 .. option:: --numParallelCollections=<int>, -j=<int>
 
    *Default*: 4
 
    Number of collections :program:`mongodump` should export
    in parallel.
 
-
 .. option:: --viewsAsCollections
 
-   When specified, :program:`mongodump` exports :manual:`read-only views
-   </core/views>` as collections. For each view, :program:`mongodump` will
-   produce a BSON file containing the documents in the view. If you
-   :binary:`~bin.mongorestore` the produced BSON file, the view will be
+   When specified, :program:`mongodump` exports :manual:`views
```

Comment:
Ref update:

:ref:`views <views-landing-page>`

NOTE: GitHub messes up the quotes for the ref.

Reply:
See prev.